config/routes.rb

```diff
@@ -1,4 +1,5 @@
 Rails.application.routes.draw do
+  devise_for :users
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
```

Comment:
Also add a root_path, which is required for devise to redirect in case user is not authenticated.

Reply:
okay